EIPS/eip-7432.md

```diff
@@ -0,0 +1,346 @@
+---
+eip: 7432
+title: Non-Fungible Token Roles
+description: Role Management for NFTs. Enables accounts to share the utility of NFTs via expirable role assignments.
+author: Ernani São Thiago (@ernanirst), Daniel Lima (@karacurt)
+discussions-to: https://ethereum-magicians.org/t/eip-7432-non-fungible-token-roles/15298
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-07-14
+requires: 165
+---
+
+## Abstract
+
+This standard introduces role management for NFTs. Each role assignment is associated with a single NFT and expires
+automatically at a given timestamp. Inspired by [ERC-5982](./eip-5982.md), roles are defined as `bytes32` and feature a
```

Comment:
Linking to another proposal means this proposal (ERC-7432) cannot advance past ERC-5982's status. So if 5982 is Review, 7432 cannot go to Last Call.

You are more than welcome to keep this link, just wanted to give you a heads up.

Reply:
Removed mentions to ERC-5982